packages/react-native-reanimated/src/logger/logger.ts

```diff
@@ -0,0 +1,75 @@
+'use strict';
+import { runOnJS } from '../threads';
+import { LogBox } from './LogBox';
+import type { LogLevel, LogData } from './LogBox';
+
+const addLog = LogBox.addLog.bind(LogBox);
```

Comment:
Since you don't use the rest of the `LogBox` here, how about exporting only `addLog` from the previous file?